src/main/CommandHandler.cpp

```diff
@@ -104,6 +104,7 @@ CommandHandler::CommandHandler(Application& app) : mApp(app)
     addRoute("generateload", &CommandHandler::generateLoad);
     addRoute("testacc", &CommandHandler::testAcc);
     addRoute("testtx", &CommandHandler::testTx);
+    addRoute("bootstrap", &CommandHandler::bootstrap);
```

Comment:
should we always expose this? This seems to be useful in general to spin up nodes, so maybe shouldn't be limited to test builds?